hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/handlers/ContractCallHandler.java

```diff
@@ -72,6 +76,12 @@ public void preHandle(@NonNull final PreHandleContext context) {
         // No non-payer signatures to verify
     }
 
+    @Override
+    public void pureChecks(@NonNull TransactionBody txn) throws PreCheckException {
+        final var op = txn.contractCallOrThrow();
+        mustExist(op.contractID(), INVALID_CONTRACT_ID);
```

Comment:
Not a blocker for this PR, but we should be consistent in the checks done in this method. We are validating `ContractID` exists in smart-contract service. But not doing any checks in Consensus service.